font/bmfont.ksy

```diff
@@ -0,0 +1,155 @@
+meta:
+  id: bmfont
+  title: BMFont binary file
+  file-extension: fnt
+  license: CC0-1.0
+  endian: le
+  bit-endian: le
```

Comment:
Usage of `bit-endian` implies that this requires KS 0.9+. Please add `ks-version: 0.9`.

Reply:
Done